modules/questions.py

```diff
@@ -63,6 +63,22 @@
 `s, get info <ADDITIONAL_INFO>` (with any filtering option mentioned so far, except `next`) can be used to get detailed information about the question as an entity in the database.
 
 ![](images/help/Questions-get-info-babyagi.png)
+
+### Reloading questions
```

Comment:
I would probably keep only 1 copy of the documentation and either link from here to README or the other way round, duplicate info might get out of sync 🤔

Reply:
Ok, I'll remove the markdown, at least for now